src/globus_cli/login_manager/tokenstore.py

```diff
@@ -160,16 +163,42 @@ def internal_auth_client():
     )
 
 
-def delete_templated_client():
-    adapter = token_storage_adapter()
-
+def delete_templated_client() -> None:
     # first, get the templated credentialed client
     ac = internal_auth_client()
 
     # now, remove its relevant data from storage
-    adapter.remove_config(_CLIENT_DATA_CONFIG_KEY)
+    remove_well_known_config("auth_client_data")
 
     # finally, try to delete via the API
     # note that this could raise an exception if the creds are already invalid -- the
     # caller may or may not want to ignore, so allow it to raise from here
     ac.delete(f"/v2/api/clients/{ac.client_id}")
+
+
+def store_well_known_config(
+    name: Literal["auth_client_data", "auth_user_data"],
+    data: dict[str, t.Any],
+    *,
+    adapter: SQLiteAdapter | None = None,
+) -> None:
+    adapter = adapter or token_storage_adapter()
+    adapter.store_config(name, data)
+
+
+def read_well_known_config(
+    name: Literal["auth_client_data", "auth_user_data"],
+    *,
+    adapter: SQLiteAdapter | None = None,
+) -> dict[str, t.Any] | None:
+    adapter = adapter or token_storage_adapter()
+    return adapter.read_config(name)
+
+
+def remove_well_known_config(
+    name: Literal["auth_client_data", "auth_user_data"],
+    *,
+    adapter: SQLiteAdapter | None = None,
+) -> None:
+    adapter = adapter or token_storage_adapter()
+    adapter.remove_config(name)
```

Comment:
Prior to these helpers which abstract away the storage adapter (which is a singleton), there were places in the code, including tests, which pulled in constants from this and one other module in order to access the config data.

On one level, that's fine, but it seemed like the need to read the user's identity ID in `globus timer create transfer` demanded a nicer interface, since the access was no longer isolated to "special" places like the `globus login` implementation.